client/src/services/dataSource.js

```diff
@@ -0,0 +1,22 @@
+import api from './api';
```

Comment:
no its not just for messages
bc we can store somethings like users too
and channels and people
another thing is this will help us a lot in the messaging part
since i dont have to worry much about rendering the socket stuff
and on app mounting
we can do an api to get everything from the db, including message history, that i can add messages to later, by just throwing it in that data store
notify the guys and call it a day